src/org/labkey/test/WebDriverWrapper.java

```diff
@@ -1458,7 +1458,7 @@ public void assertTextPresent(TextSearcher searcher, String... texts)
 
         if (!missingTexts.isEmpty())
         {
-            String failMsg = (missingTexts.size() == 1 ? "Text '" : "Texts ['") + String.join("', '", missingTexts) + "'";
+            String failMsg = (missingTexts.size() == 1 ? "Text '" : "Texts ['") + String.join("','", missingTexts) + "'";
```

Comment:
Is this change necessary?

Reply:
It's not.. Just keeping the fail msg consistent with the text.

Reply:
I see, this is used by many places. I will update.